### ISSUE TITLE
ModuleNotFoundError: No module named 'termios'         

### ISSUE THREAD

**Issue by hadifar** [meta-llama/llama-models#144]
I followed the installation instruction using pip, however after running the following command

`llama model list --show-all`

 it says:

```
Traceback (most recent call last):                                                                                                         
  File "\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main                           
    return _run_code(code, main_globals, None,                                                                                             
  File "\Python\Python310\lib\runpy.py", line 86, in _run_code                                      
    exec(code, run_globals)                                                                                                                
  File "\llama.exe\__main__.py", line 4, in <module>                            
    from llama_toolchain.cli.llama import main                                                                                             
  File "\llama_toolchain\cli\llama.py", line 11, in <module>          
    from .stack import StackParser                                                                                                         
  File "C:\Users\hadifar\IdeaProjects\function-calling\venv\lib\site-packages\llama_toolchain\cli\stack\__init__.py", line 7, in <module>  
    from .stack import StackParser  # noqa                                                                                                 
  File "llama_toolchain\cli\stack\stack.py", line 12, in <module>    
    from .configure import StackConfigure                                                                                                  
  File "\llama_toolchain\cli\stack\configure.py", line 19, in <module>
    from llama_toolchain.common.exec import run_with_pty                                                                                   
  File "llama_toolchain\common\exec.py", line 9, in <module>         
    import pty                                                                                                                             
  File "Python310\lib\pty.py", line 12, in <module>                                         
    import tty                                                                                                                             
  File "tty.py", line 5, in <module>                                          
    from termios import *                                                                                                                  
ModuleNotFoundError: No module named 'termios'             
```

OS: Windows 10 
Python: 3.10.10

**Comment by ashwinb** [comment]
What version of the `llama-toolchain` or `llama-stack` package did you install?

**Comment by wmmc88** [comment]
Looks like the offending line is https://github.com/meta-llama/llama-stack/blob/e6fdb9df29de6b1723404de355801e8ec455b8a1/llama_stack/distribution/utils/exec.py#L14C1-L14C8. `termios` is not available for use on windows. How are windows users intended to download models if `llama-toolchain` doesn't work on windows?

**Comment by ashwinb** [comment]
@wmmc88 
> How are windows users intended to download models if llama-toolchain doesn't work on windows?

yeah we will fix this. for now, we are moving around our imports so anyone who isn't using the full functionality of the llama CLI isn't paying for this cost and so `llama model` will work. but the proper fix is also to make sure we have some replacement for termios so all of the llama CLI works on Windows

**Comment by ashwinb** [comment]
This should be fixed by llama-stack>=0.0.20. Re-open if not.

**Comment by sinansonmez** [comment]
I am still gettin the issue on v0.0.63
